recipe/meta.yaml

```diff
@@ -35,7 +35,7 @@ requirements:
     - numpy >=1.23.5
     - salib >=1.4
     - seaborn
-
+    - pandas <=2.1.4
```

Comment:
Minor hint: Try to sort the requirements and leave an empty line to the next top level category of the yaml

Reply:
Thanks! I've kept python at the top, if that's okay.